### PR TITLE
Task-26-For-Testing

### PR DESCRIPTION
**Pull request by hnbmunoz**

### CODE REVIEW

File: react-client/src/Components/App.js [hunk 1]
```diff
@@ -46,13 +46,20 @@ function App() {
       if (e.target.id !== "closedompeeker" && domSwitch == true) {
         e.preventDefault();
 
+        const clsArr =[...e.target.classList].map((cls) => {
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: You can do this instead: 
```suggestion
        [...e.target.classList].map((cls) => ({ clsName: `.${cls}` });
```

File: react-client/src/Components/DomInfoDialogBox.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { useeffect, useState } from "react";
 
-const DomInfoDialogBox = ({ id,idx,clstag,clsname,parenttag,parentId,parentClass,children,top, left, onClose,fontsize, fontfamily,textcolor
+const DomInfoDialogBox = ({ id,idx,clstag, clsname,parenttag,parentId,parentClass,children,top, left, onClose,fontsize, fontfamily,textcolor
```
Comment (reciosonny) [inline react-client/src/Components/DomInfoDialogBox.js]: Be consistent with variable spacing please.

Comment (reciosonny) [reply react-client/src/Components/DomInfoDialogBox.js]: They're inconsistent.

File: react-client/src/Components/App.js [hunk 3]
```diff
@@ -70,10 +77,11 @@ function App() {
             y: e.pageY,
             id: e.target.id.trim() !== "" ? "#" + e.target.id.trim() : null,
             clstag: e.target.localName,
-            clsname:
-              e.target.className.trim() !== ""
-                ? "." + e.target.className.trim()
-                : null,
+            // clsname:
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: Remove commented out code

File: react-client/src/Components/DomInfoDialogBox.js (hunk 2 above)
Comment (reciosonny) [reply react-client/src/Components/DomInfoDialogBox.js]: Always use space after comma like so: `id, idx, clstag`

Comment (hnbmunoz) [reply react-client/src/Components/DomInfoDialogBox.js]: sorry will correct had to edit during testing

**Review by reciosonny — APPROVED** [review]

**Comment by reciosonny** [comment]
Okay looks good.

**Pull request merged**